service/topology_coordinator.cc

```diff
@@ -1053,7 +1053,7 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
                 case locator::tablet_transition_stage::allow_write_both_read_old:
                     if (action_failed(tablet_state.barriers[trinfo.stage])) {
                         if (check_excluded_replicas()) {
-                            transition_to_with_barrier(locator::tablet_transition_stage::revert_migration);
+                            transition_to_with_barrier(locator::tablet_transition_stage::cleanup_target);
```

Comment:
@tgrabiec should we land this patch independently of this series?

Reply:
It's not a fix, but a change which is needed by this PR, so I don't think we need to.